app/org/sagebionetworks/bridge/play/controllers/ReportController.java

```diff
@@ -45,10 +50,21 @@ final void setAccountDao(AccountDao accountDao) {
     }
     
     /**
-     * Reports for specific individuals accessible only to consented study participants.
+     * Get a list of the identifiers used for participant reports in this study.
+     */
+    public Result getParticipantReportIndices() throws Exception {
+        UserSession session = getAuthenticatedAndConsentedOrResearcherSession();
+        
+        List<? extends ReportIndex> indices = reportService.getReportIndices(
+                session.getStudyIdentifier(), ReportType.PARTICIPANT);
+        return okResult(indices);
+    }
+    
+    /**
+     * Reports for specific individuals accessible to either consented study participants, or to researchers.
      */
     public Result getParticipantReport(String identifier, String startDateString, String endDateString) {
-        UserSession session = getAuthenticatedAndConsentedSession();
+        UserSession session = getAuthenticatedAndConsentedOrResearcherSession();
```

Comment:
This method gets the report for the current user's health code, which means even though Researchers can access this, they can't do anything useful with it.